Installation instructions are wrong

Following a fresh clone, `make install` fails with
```sh
➜  ft-origami git:(gh-pages) make install
Makefile:6: node_modules/@financial-times/origami-service-makefile/index.mk: No such file or directory
make: *** No rule to make target `node_modules/@financial-times/origami-service-makefile/index.mk'.  Stop.
```

This is due to the fact that the Makefile immediately expects there to be a `node_modules` directory (lol)

https://github.com/Financial-Times/ft-origami/blob/e6682e09d923394f93eec8cae0a133ef041bcda0/Makefile#L1-L8


Not sure what the preference would be your end. Update docs? Change the Makefile? Whatevs

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ This is the source code for the Origami website. Please visit [http://origami.ft
 Now install all the dependencies required to build the site:
 
 ```sh
-make install
+npm install && make install
 ```
 
 After this, you can start running the site using:

